pywinauto/unittests/test_UIAWrapper.py

```diff
@@ -40,10 +40,9 @@ class UIAWrapperTests(unittest.TestCase):
         def setUp(self):
             """Start the application set some data and ensure the application
             is in the state we want it."""
-            backend.activate("uia")
 
             # start the application
-            self.app = Application().Start(os.path.join(wpf_samples_folder, u"WpfApplication1.exe"))
+            self.app = Application(backend = 'uia').Start(os.path.join(wpf_samples_folder, u"WpfApplication1.exe"), timeout = 30)
```

Comment:
I see a lot of long lines like that. Why just don't split it:
```python
path_to_exe = os.path.join(wpf_samples_folder, u"WpfApplication1.exe")
self.app = Application(backend = 'uia').Start(path_to_exe, timeout = 30)
```
Even PEP-8 talks about 79 chars per line.